Sass/style.css

```diff
@@ -0,0 +1,555 @@
+}
+
+.nav-bar {
+  background-color: rgb(246, 238, 238);
+  box-shadow: 0 2px 8px rgba(10, 9, 9, 0.26);
+  position: sticky;
+  display: block;
+  justify-content: space-between;
+  width: 100%;
+  min-height: 60px;
+}
+
+#nav-menu-cc {
+  border: 5px solid #ec5242;
+  padding: 8px;
+  width: 25%;
+  color: #ec5242;
+}
+
+.nav-bar .logo {
+  padding-left: 10px;
+  color: black;
+  font-weight: bolder;
+  font-size: 40px;
+}
+
+.nav-bar ul {
+  display: none;
+}
+
+.nav-bar button {
+  width: -moz-fit-content;
+  width: fit-content;
+  float: right;
+  border: none;
+  margin: 5px 10px 0 0;
+  background-color: transparent;
+}
+
+.courses img {
+  border-radius: 10px;
+  height: 60px;
+  width: 60px;
+  margin-bottom: 10px;
+}
+
+.nav-bar button img {
+  width: 50px;
+  height: 50px;
+}
+
+.home-page {
+  display: flex;
+  flex-direction: column;
+  min-height: 600px;
+  background-image: url(../images/main-background-img.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.home-page h1 {
+  color: #ec5242;
+  font-size: 50px;
+  font-family: "COCOGOOSE", sans-serif;
+  font-weight: 400;
+  text-transform: uppercase;
+}
+
+.home-page h1 strong {
+  color: #ec5242;
+  font-size: 40px;
+}
+
+.home-page h1 span {
+  color: #ec5242;
+  font-size: 20px;
+}
+
+.home-page .home-buttons-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+
+.home-page .home-buttons-div button {
+  margin-top: 15px;
+  width: 90%;
+  min-height: 60px;
+  text-align: center;
+  font-size: 20px;
+  color: rgb(255, 255, 255);
+  background-color: rgb(255, 0, 0);
+  border-radius: 25px;
+  margin-bottom: 20px;
+}
+
+.copy {
+  margin-top: 140px;
+  float: left;
+  font-weight: 700;
+}
+
+.copy b {
+  font-size: 30px;
+}
+
+.about-us {
+  display: flex;
+  flex-direction: column;
+  padding-top: 30px;
+  align-items: center;
+  background-image: url(../images/abt-bg.jpg);
+  width: 100%;
+}
+
+.about-us p {
+  display: flex;
+  width: 100%;
+  padding: 10px;
+  margin-top: 30px;
+  margin-bottom: 80px;
+  background-color: #fff;
+  box-shadow: 0 2px 8px rgba(104, 102, 102, 0.895);
+  min-height: 100px;
+}
+
+.about-us h1 {
+  display: flex;
+  float: left;
+  font-size: 50px;
+  color: #ec5242;
+}
+
+.about-us-descrptn {
+  display: flex;
+  justify-content: center;
+  background-color: white;
+  padding-left: 10px;
+  padding-bottom: 10px;
+  height: auto;
+}
+
+.courses p {
+  display: flex;
+  width: 85%;
+  color: white;
+}
+
+.about-us-descrptn .inner p {
+  display: flex;
+  width: 90%;
+}
+
+.joinUs-section h2 {
+  display: flex;
+  color: white;
+}
+
+.about-us-descrptn .inner h2 {
+  display: flex;
+
+  /* font-size: 20px; */
+}
+
+.courses-section {
+  display: flex;
+  padding-top: 20px;
+  flex-direction: column;
+  color: #fff;
+  justify-content: center;
+  align-items: center;
+  background-color: #212529;
+  width: 100%;
+}
```

Comment:
Great design @degisew 🤝

- [ ] As pointed out by the previous reviewers, kindly make sure your final design is a replica of the design given. Take a closer look at the details of this section and make changes accordingly
- Your design
![page-2](https://user-images.githubusercontent.com/63917725/183267126-91bd3917-e915-437e-a35c-6b592436b59c.PNG)

- Given
![page2-design](https://user-images.githubusercontent.com/63917725/183267141-86b39779-aa2f-45ab-b8cf-e3496ec769f4.PNG)
 design